includes/admin/settings/display-settings.php

```diff
@@ -12,6 +12,18 @@
 // Exit if accessed directly
 if ( ! defined( 'ABSPATH' ) ) exit;
 
+function edd_admin_header() {
+	if ( ! edd_is_admin_page( '', '', false ) ) {
+		return;
+	}
+	?>
+	<div id="edd-header" class="edd-header">
+		<img class="edd-header-logo" alt="Easy Digital Downloads logo" src="<?php echo EDD_PLUGIN_URL . '/assets/images/logo-edd-dark.svg'; ?>" />
```

Comment:
```suggestion
		<img class="edd-header-logo" alt="" src="<?php echo EDD_PLUGIN_URL . '/assets/images/logo-edd-dark.svg'; ?>" />
```
This is a decorative image.